eureka-webapp/src/main/webapp/index.html

```diff
@@ -138,10 +152,10 @@
 
         <!-- build:js vendor.min.js -->
         <!-- bower:js -->
-        <script src="bower_components/jquery/dist/jquery.js"></script>
         <script src="bower_components/angular/angular.js"></script>
         <script src="bower_components/ui-router/release/angular-ui-router.js"></script>
         <script src="bower_components/angular-messages/angular-messages.js"></script>
+        <script src="bower_components/jquery/dist/jquery.js"></script>
```

Comment:
Help me understand the change in load order. We loaded jquery before angular to prevent angular from trying to use the built-in jquery lite. I believe the original load order is correct. The answer to the question, "Does AngularJS use the jQuery library?", in the Angular FAQ (https://docs.angularjs.org/misc/faq) seems to back me up.